TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/DemoTeleOp.kt

```diff
@@ -0,0 +1,35 @@
+package org.firstinspires.ftc.teamcode.opmode.teleop
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp
+import org.firstinspires.ftc.teamcode.api.DemoAPI
+import org.firstinspires.ftc.teamcode.api.linear.DemoLinearAPI
+
+@TeleOp(name = "Demo TeleOp")
```

Comment:
```suggestion
@TeleOp(name = "Demo TeleOp", group = "Demo")
```

Also same thing as above :)